public/app/features/explore/Logs/Logs.tsx

```diff
@@ -602,16 +603,21 @@ class UnthemedLogs extends PureComponent<Props, State> {
         </PanelChrome>
         <PanelChrome
           titleItems={[
-            config.featureToggles.logsExploreTableVisualisation ? (
-              this.state.visualisationType === 'logs' ? null : (
-                <PanelChrome.TitleItem title="Experimental" key="A">
-                  <FeatureBadge
-                    featureState={FeatureState.beta}
-                    tooltip="This feature is experimental and may change in future versions"
-                  />
-                </PanelChrome.TitleItem>
-              )
-            ) : null,
+            config.featureToggles.logsExploreTableVisualisation
+              ? this.state.visualisationType === 'logs'
+                ? null
+                : [
+                    <PanelChrome.TitleItem title="Experimental" key="A">
+                      <FeatureBadge
+                        featureState={FeatureState.beta}
+                        tooltip="This feature is experimental and may change in future versions"
```

Comment:
```suggestion
                        tooltip="Table view is experimental and may change in future versions"
```

Not related to this PR, but now, when we always have **Logs** title, I think it makes it more clear what do we mean by `beta feature` and what feedback are we looking for. Btw it would be cool if `<FeatureBadge` could accept feedbackLink and would handle creating the feedback component - in the future. 🙂